core/src/jws.rs

```diff
@@ -0,0 +1,84 @@
+use crate::{Base64String, Base64UrlString, DidDocument, Jwk, MultiBase32String};
+use cid::Cid;
+use serde::{Deserialize, Serialize};
+use ssi::jwk::Algorithm;
+
+#[derive(Serialize, Deserialize)]
+pub struct JwsSignature {
+    pub protected: Option<Base64String>,
+    pub signature: Base64UrlString,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct Jws {
```

Comment:
I'd like to understand what needs to change in the `dag-jose` crate in order to be able to use its types here?

I understand this `Jws` struct also add the signing/verification logic however I do not see the `dag-jose` crate used anywhere in the code base. As such seems like we should fix that crate to make it amenable to this code as its purpose was to be a building block for Ceramic events.

Reply:
Yeah, I was really surprised once I got everything implemented, that the events we pass to ceramic don't actually use dag-jose. They just have the data so that ipfs can encode them as dag-jose.

Seems like there's a mismatch there.